tests/container_disk_test.go

```diff
@@ -142,7 +142,7 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting with virtio-win", func() {
 		Context("with virtio-win as secondary disk", func() {
-			It("[test_id:1467]should boot and have the virtio as sata CDROM", func() {
+			It("[test_id:1467]should boot and have the virtio as sata CDROM", decorators.Conformance, func() {
```

Comment:
I think this is just a variation, what bus I can ask for... I would not include it.

Reply:
Sure! Done